web/wp-content/themes/dctx-v2/template-parts/content-blocks/block-wds-related-posts.php

```diff
@@ -0,0 +1,82 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-eq6whm.php, but found phpcs-scan-eq6WHm.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).